auth/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,75 @@
+    <string name="resend_code_in" translation_description="Countdown timer text that the user needs to wait for before attempting to resend verification sms">Code opnieuw verzenden over 0:%02d</string>
+    <string name="verify_your_phone_title" translation_description="Button text to complete phone number verification">Uw telefoonnummer verifiëren</string>
+    <string name="verifying" translation_description="Progress dialog text while phone number is being verified">Verifiëren…</string>
+    <string name="retrieving_sms" translation_description="Progress dialog text when sms is being retrieved before auto-submit">Sms ophalen…</string>
+    <string name="incorrect_code_dialog_body" translation_description="Inline error when incorrect sms verification code is being used to verify">Onjuiste code. Probeer het opnieuw.</string>
+    <string name="incorrect_code_dialog_positive_button_text" translation_description="Text to dismiss error alert dialog">OK</string>
+    <string name="error_too_many_attempts" translation_description="Error message when the phone number has been used too many times">Dit telefoonnummer is te vaak gebruikt</string>
+    <string name="error_quota_exceeded" translation_description="Error message when the Firebase project's quota has been exceeded.">Er is een probleem met de verificatie van uw telefoonnummer</string>
+    <string name="error_session_expired" translation_description="Error message when SMS confirmation code has expired.">Deze code is niet meer geldig</string>
+    <string name="sign_in_with_phone_number" translation_description="Label for phone number input form">Inloggen met telefoonnummer</string>
+    <string name="done" translation_description="Progress dialog success transition once sms is sent">Gereed</string>
+    <string name="verified" translation_description="Progress dialog success transition when phone number was verified">Geverifieerd</string>
+    <string name="code_sent" translation_description="Progress dialog success transition once sms is sent">Code verzonden</string>
+    <string name="resending" translation_description="Progress dialog text when user clicks on resend sms">Opnieuw verzenden…</string>
+    <string name="resend_code" translation_description="Link text to resend verification sms">Code opnieuw verzenden</string>
+    <string name="verify_phone_number" translation_description="Button text to submit confirmation code and complete phone verification">Telefoonnummer verifiëren</string>
+    <string name="continue_phone_login" translation_description="Button text to submit phone number and send sms">Doorgaan</string>
+    <string name="sms_terms_of_service" translation_description="Fine print warning displayed below Verify Phone Number button">Als u op \'%1$s\', ontvangt u mogelijk een sms. Er kunnen sms- en datakosten in rekening worden gebracht.</string>
```

Comment:
A verb is missing in this sentence. It should be `Als u op \'%1$s\' tikt, ontvangt u mogelijk een sms. Er kunnen sms- en datakosten in rekening worden gebracht.`